<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>960</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageLoading">
     <widget class="QLabel" name="loadingBG">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="loadingBtn">
      <property name="geometry">
       <rect>
        <x>753</x>
        <y>372</y>
        <width>125</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLogin">
     <widget class="QPushButton" name="loginBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>351</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="autocheck">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
 padding-left: 4px;
   color: white;
}
QCheckBox::indicator {
     width: 16px;
     height: 16px;
 }
</string>
         </property>
         <property name="text">
          <string> 자동 로그인</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   color: white;
}</string>
           </property>
           <property name="text">
            <string>아이디 찾기</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{

   color: white;
}</string>
           </property>
           <property name="text">
            <string>비밀번호 찾기</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{

   color: white;
}</string>
           </property>
           <property name="text">
            <string>회원가입</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>351</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="inputId_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputPw_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="loginBG">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="img/img.qrc">
        <normaloff>:/left-arrow.png</normaloff>:/left-arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <zorder>loginBG</zorder>
     <zorder>loginBtn</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>backBtn2</zorder>
    </widget>
    <widget class="QWidget" name="pageSingIn">
     <widget class="QLabel" name="signinBG">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUpEmail_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>349</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUpNickname_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>349</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>NickName</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUpPwCheck_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>349</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Password Check</string>
      </property>
     </widget>
     <widget class="QPushButton" name="requestSignUpBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>349</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>다음</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUpPw_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>349</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="img/img.qrc">
        <normaloff>:/left-arrow.png</normaloff>:/left-arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageGame">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="pageRank">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>891</width>
        <height>361</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Score</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Datetime</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>881</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>테트리스 랭킹</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="img/img.qrc">
        <normaloff>:/left-arrow.png</normaloff>:/left-arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="rankBG">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>960</width>
        <height>540</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>30</y>
        <width>89</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>랭크 갱신</string>
      </property>
     </widget>
     <zorder>rankBG</zorder>
     <zorder>tableWidget</zorder>
     <zorder>label</zorder>
     <zorder>backBtn4</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
